core/src/components/button/button.md.scss

```diff
@@ -160,3 +160,12 @@
     background: #{current-color(base)};
   }
 }
+
+
+// Activated Button in Toolbar
+// --------------------------------------------------
+
+:host(.button-outline.ion-activated.in-toolbar:not(.ion-color):not(.in-toolbar-color)) .button-native {
```

Comment:
Let's add a comment that this needs to be within this file instead of `button.scss` else it gets overridden. That way we don't end up moving it back.

Reply:
Wouldn't that be the case for all the code in these mode-specific files? Is there something about these styles that seem like they'd be more likely to be moved back by accident?

Note that the MD/iOS styles are slightly different now, so putting them in the mode-specific files isn't a hack or anything.

Reply:
Oh, I see. Then this probably ties to my question to the [iOS background style](https://github.com/ionic-team/ionic-framework/pull/29216#discussion_r1548321998).

Reply:
I'm good without the comments, thank you!

Reply:
You're good to merge.